go.mod

```diff
@@ -4,7 +4,7 @@ go 1.21
 
 require (
 	github.com/CosmWasm/wasmvm/v2 v2.0.0
-	github.com/cosmos/cosmos-proto v1.0.0-beta.4
+	github.com/cosmos/cosmos-proto v1.0.0-beta.5
 	github.com/cosmos/cosmos-sdk v0.50.5
```

Comment:
SDK v0.50.6 was just released. Do we want this in here as well?

Reply:
yes, I will create a new pr for it..